source/puppet/4.9/style_guide.markdown

```diff
@@ -502,15 +502,15 @@ file {
     mode   => '0666' ;
  
   '/foo' :
-    user => root;
+    user => 'root';
  
   '/bar' :
-    user => staff;
+    user => 'staff';
 }
  
 # Give the defaults a name if used several times
 $our_default_file_attributes = { 
-  'mode' => '0666', 
+  'mode'   => '0666', 
   'ensure' => 'file', 
```

Comment:
@sammcj , this is our error, not yours, but if you make changes and want to address, it's appreciated.